tcl/tools/analysis.tcl

```diff
@@ -3186,7 +3187,8 @@ proc setAutomoveTime {{n 1}} {
     pack $w.f -expand 1
     ttk::label $w.f.label -text "Set the engine thinking time per move in seconds:"
     pack $w.f.label -side top -pady 5 -padx 5
-    ttk::entry $w.f.entry -background white -width 10 -textvariable temptime
+    ttk::spinbox $w.f.entry -width 5 -textvariable temptime -from 0.5 -to 999 -increment 0.1 \
```

Comment:
Same spinbox problems

Reply:
listbox is replaced with treeview. Sorting columns with single click on header. maybe an indicator, which column is sorted could be added. But is very obvious which is the sort column
spinboxes: there a two validations (trace and validcommand). it seems it is sufficent to have the "trace-command" for evaluation the format. revert spinbox increment to 1